docs/roles/add-role.md

```diff
@@ -20,5 +20,7 @@ links:
 1. Optionally type a `Description` explaining the purpose of the role
 1. Click `NEXT`
 1. In the `Permissions` step select the permissions for each resource type
-    > Make sure the `VIEW` permission in the resource type `SERVER` is marked, otherwise the team member will not have direct access to the environment through the web app.
 1. Click `CONFIRM`
+    - In the `Roles` list, the recently created `Role` can be seen
+    - A Team is required for a user to use this role's permissions in a environment's resources
```

Comment:
If the add team membership to an environment already requires a team and a role, don't think here we are adding too much information for a reader that just wants to know how to add a role?

This kinda `what's next?` info should be handled by the UX flow in the web app UI and not added to every doc. If we add every info like this to the documentation this  will require more work to keep our documentation up to date with UX changes.

My take is: the person came to add a role, we instruct how to add a role, period. The `next` attribute should take the person to next step, so there is no need to elaborate on what's next.

Thoughts?

Reply:
Makes sense, articles updated